docs/advanced-guide/extending-muse.ipynb

```diff
@@ -89,7 +89,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from typing import List, Optional, Text\n",
+    "from typing import Optional\n",
```

Comment:
I guess we have to wait for a newer version of Python until we can write `Optional`s as `my_type | None`!